website/docs/cloud-docs/projects/manage.mdx

```diff
@@ -0,0 +1,119 @@
+team access to the project.
+
+## Automatically destroy inactive workspaces 
+
+<!-- BEGIN: TFC:only name:pnp-callout -->
+@include 'tfc-package-callouts/ephemeral-workspaces.mdx'
+<!-- END: TFC:only name:pnp-callout -->
+
+You can configure HCP Terraform to automatically destroy each workspace's
+infrastructure in a project after a period of inactivity. A workspace
+is _inactive_ if the workspace's state has not changed within your designated
+time period.
+
+If you configure a project to auto-destroy its infrastructure when inactive, 
+any run that updates Terraform state further delays the scheduled auto-destroy 
+time by the length of your designated timeframe.
+
+!> **Note:** Automatic destroy plans _do not_ prompt you for apply approval in the HCP Terraform user interface. We recommend only using this setting for development environments.
+
+To schedule an auto-destroy run after a period of workspace inactivity:
+
+1. Navigate to the project's **Settings** > **Auto-destroy Workspaces** page.
+1. Click **Set up default**.
+1. Select or customize a desired timeframe of inactivity.
+1. Click **Confirm default**.
+
+You can configure an individual workspace's auto-destroy settings to override
+this default configuration. Refer to [automatically destroy workspaces](/terraform/cloud-docs/workspaces/settings/deletion#automatically-destroy) for more information.
+
+## Delete a project
+
+You can only delete projects that do not contain workspaces.
+
+To delete an empty project:
+
+1. Click **Projects**.
+1. Search for a project that you want to review by scrolling down the table or
+   searching for a project name in the search bar above the project table.
+1. Click **Settings**. The settings view for the selected project appears.
+1. Click the **Delete** button. A **Delete project** modal appears.
+1. Click the **Delete** button to confirm the deletion.
+
+HCP Terraform returns to the **Projects** view with the deleted project
+removed from the list.
+
+## Define project tags
+You can define tags stored as key-value pairs to help you organize your projects and track resource consumption. HCP Terraform applies tags that you attach to projects to the workspaces created inside the project.
+
+Workspace administrators with appropriate permissions can attach new key-value pairs to their workspaces to override inherited tags. Refer to [Workspace tags](/terraform/cloud-docs/workspaces/create#define-workspace-tags) for additional information about using tags in workspaces.
+
+Tags that you create appear in the tags management screen in the organization settings. Refer to [Organizations](/terraform/cloud-docs/users-teams-organizations/organizations) for additional information.
+
+You cannot override reserved tag keys created by the organization administrator. Refer to [Create and manage reserved tags](/terraform/cloud-docs/users-teams-organizations/organizations#create-and-manage-reserved-tags) for additional information.
```

Comment:
I think this stipulation only applies to workspaces -- which is the only resource where you can both inherit and define tags.